frontend/components/PerceivedDemographicsContainer.jsx

```diff
@@ -20,6 +21,7 @@ const mapStateToProps = state => ({
   questionOrder: demographicsOrder(state),
   current: selectedWorkloadItemIndex(state),
   total: totalWorkloadItemCount(state),
+  workload: selectWorkload(state),
```

Comment:
With the change to the sagas file, these additions to this container are not necessary